ETLChemistryAnalyzer/src/org/labkey/ETLChemistryAnalyzer/ETLChemistryAnalyzerModule.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2015 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.labkey.ETLChemistryAnalyzer;
+
+import org.jetbrains.annotations.NotNull;
+import org.labkey.api.data.Container;
+import org.labkey.api.data.ContainerManager;
+import org.labkey.api.module.DefaultModule;
+import org.labkey.api.module.ModuleContext;
+import org.labkey.api.view.WebPartFactory;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Set;
+
+public class ETLChemistryAnalyzerModule extends DefaultModule
+{
+    public static final String NAME = "ETLChemistryAnalyzer";
+
+    @Override
+    public String getName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public double getVersion()
+    {
+        return 16.21;
```

Comment:
What version number should you recommend to use for this module that we have develop and maintain in our own. We could start with 20.001, or stay with 16.021?

Reply:
For modules that aren't shared with anyone else, it doesn't matter too much. If you want to be consistent with the core modules, that's great. Otherwise, you can choose whatever numbering approach you want, subject to the SQL upgrade script running rules.

https://www.labkey.org/Documentation/wiki-page.view?name=sqlScripts#ver